docs/sources/reference/api/registry_api.md

```diff
@@ -503,7 +503,12 @@ Status Codes:
 
 ## Search
 
-If you need to search the index, this is the endpoint you would use.
+If you need to search the index, this is the endpoint you would use. Note that if the request's
```

Comment:
If it's a note format as a note - https://github.com/docker/docker/blob/master/docs/sources/contributing/docs_style-guide.md#notes